### PR TITLE
KAR-717 fix

### PR DESCRIPTION
**Pull request by garima468**
KAR-717 fix

### CODE REVIEW

File: project/ws/app/src/lib/routes/home/components/user-cards/user-card.component.html [hunk 1]
```diff
@@ -403,9 +402,9 @@
           <div class="flex flex-row margin-top-m">
             <div class="md:w-1/3">
               <label class="view-text-label">Group</label>
-              <p>{{user.profileDetails && user.profileDetails.professionalDetails &&
-                user.profileDetails.professionalDetails[0].group ?
-                user.profileDetails.professionalDetails[0].group : '-'}}
+              <p>{{user?.profileDetails && user?.profileDetails?.professionalDetails &&
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/home/components/user-cards/user-card.component.html]: please put condition in bracket

File: project/ws/app/src/lib/routes/home/components/user-cards/user-card.component.html [hunk 2]
```diff
@@ -420,9 +419,9 @@
             </div>
             <div class="md:w-1/3">
               <label class="view-text-label">Designation</label>
-              <p>{{user.profileDetails && user.profileDetails.professionalDetails &&
-                user.profileDetails.professionalDetails[0].designation ?
-                user.profileDetails.professionalDetails[0].designation : '-'}}
+              <p>{{user?.profileDetails && user?.profileDetails?.professionalDetails &&
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/home/components/user-cards/user-card.component.html]: same as above

File: project/ws/app/src/lib/routes/home/components/user-cards/user-card.component.html [hunk 3]
```diff
@@ -733,15 +733,15 @@
...
+                <p>{{appData?.user?.profileDetails && appData?.user?.profileDetails?.employmentDetails &&
+                  appData?.user?.profileDetails?.employmentDetails?.pinCode ?
                   appData.user.profileDetails.employmentDetails.pinCode : '-'}}</p>
               </div>
 
               <div class="md:w-1/4">
                 <label class="view-text-label">eHRMS ID</label>
-                <p>{{appData.user.profileDetails && appData.user.profileDetails.additionalProperties &&
-                  appData.user.profileDetails.additionalProperties.externalSystemId ?
+                <p>{{appData?.user?.profileDetails && appData?.user?.profileDetails?.additionalProperties &&
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/home/components/user-cards/user-card.component.html]: same as above

File: project/ws/app/src/lib/routes/home/components/user-cards/user-card.component.html (hunk 1 above)
Comment (garima468) [reply project/ws/app/src/lib/routes/home/components/user-cards/user-card.component.html]: done

File: project/ws/app/src/lib/routes/home/components/user-cards/user-card.component.html (hunk 2 above)
Comment (garima468) [reply project/ws/app/src/lib/routes/home/components/user-cards/user-card.component.html]: done

**Pull request merged**